pkg/network.go

```diff
@@ -175,6 +175,11 @@ const (
 	// already using labels for domain, it probably best to keep this
 	// consistent.
 	VisibilityLabelKey = "networking.knative.dev/visibility"
+
+	// DirectLoadbalancingHeaderName is the name of a header that signifies to
+	// loadbalancers that the respective request should not be loadbalanced again but
+	// sent to the request's target directly.
+	DirectLoadbalancingHeaderName = "K-Direct-Lb"
```

Comment:
```suggestion
	// DirectLoadbalancingHeaderName is the name of the header that directs
	// load balancers that the respective request should not be load balanced but
	// sent to the request's target directly.
	DirectLoadbalancingHeaderName = "K-Direct-Lb"
```

I've edited the text, how I'd have written it, but you can ignore some or all of it :-)

Finally, should we just call it `passthrough` rather than `direct`?

Reply:
Done.